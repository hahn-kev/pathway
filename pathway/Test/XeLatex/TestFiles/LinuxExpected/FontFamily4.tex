\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\ta="Charis SIL":color=ff0000 at 20pt
\font\tbta="Charis SIL":color=0000ff at 16pt
\font\tctbta="Charis SIL":color=008000 at 32pt

\pagestyle{fancy} 

\ta{Ta class }

\tbta{Tb class }

\tctbta{Tc class }

\tbta{Tb class }

\ta{Ta class }

\end{document}
