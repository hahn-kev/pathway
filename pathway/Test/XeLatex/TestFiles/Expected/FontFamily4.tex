\documentclass[a4paper]{article} 

\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[margin=2cm,includeheadfoot]{geometry}
\begin{document} 
\pagestyle{plain} 
\font\ta="Times New Roman":color=ff0000 at 20pt
\font\tbta="Arial":color=0000ff at 16pt
\font\tctbta="Times New Roman":color=008000 at 32pt

\pagestyle{fancy} 

\ta{Ta class }

\tbta{Tb class }

\tctbta{Tc class }

\tbta{Tb class }

\ta{Ta class }

\end{document}
